src/components/checkbox/checkbox.scss

```diff
@@ -12,6 +12,12 @@ $checkbox-top: 12px !default;
   }
 }
 
+// Needed for check to display correctly in IE11
```

Comment:
Added some more detail to the comment. Let me know if you would like it refined further.